helm_deploy/hmpps-interventions-service/reports/team_kpi_details_counts.sql

```diff
@@ -4,8 +4,12 @@ SELECT ct.name
      , COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL )                            AS received
      , COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL AND r.concluded_at ISNULL )  AS in_progress
      , COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL AND r.concluded_at NOTNULL ) AS finished
-     , ROUND(100.0 * COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL AND r.concluded_at ISNULL ) /
-             COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL ), 2)                  AS in_progress_percent
+     , CASE WHEN COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL ) > 0
+     	   THEN ROUND(100.0 * COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL AND r.concluded_at ISNULL ) /
+	             COUNT(r.id) FILTER ( WHERE r.sent_at NOTNULL ), 2)       
+	   ELSE 0.00
+       END
+                                                                                 AS in_progress_percent
```

Comment:
Nice 👍 

To be honest, I never heard any feedback on `in_progress_percent`, so we can also remove it if that's simpler